pkg/backends/cloudwatch/cloudwatch.go

```diff
@@ -21,34 +22,52 @@ const MAX_DIMENSIONS = 10
 // BackendName is the name of this backend.
 const BackendName = "cloudwatch"
 
+type QueueItem struct {
+	metrics  []*cloudwatch.MetricDatum
+	callback *gostatsd.SendCallback
```

Comment:
Is there any reason for this to be a pointer?  It makes the actual invocation rather ugly, and I don't see any actual need for it? (although I might be missing something)